setup.py

```diff
@@ -8,6 +9,17 @@
 with open('README.md', 'r') as f:
     readme = f.read()
 
+
+if os.system('curl --version | grep NSS 2>/dev/null') != 0:
+    os.environ['PYCURL_SSL_LIBRARY'] = 'openssl'
```

Comment:
here 'NSS' should be there .. .as ur greping for nss.

Reply:
It is fine as exp has != 0.